### DISCUSSION TITLE
Browsers table questions

### DISCUSSION THREAD

**Discussion by NansySalem** [browserslist/browserslist#769] [General]
I am trying to figure out the minimum versions my app should support for Edge and Chrome in Mobile. I am a bit confused by the browsers table link [here](https://github.com/browserslist/browserslist#browsers). 

1- For Edge and Chrome in Android, is it safe to assume that I can look at the support for `Chrome for Android` in caniuse?
2-  For Edge and Chrome in iOS, is it safe to assume that I can look at the support for `Safari on iOS`  in caniuse?
   - Should I be providing an edge/chrome version or a safari version?
   - Is there internal mapping, for example if I'd like my app to support an older Edge version where mobile versioning was different from desktop, will this automatically map to a safari version automatically and vice versa?
   
3- What is `ChromeAndroid` and how it is different from `and_chr`

**Comment by ai** [comment]
> What is ChromeAndroid and how it is different from and_chr

It is just an alias

>  For Edge and Chrome in iOS, is it safe to assume that I can look at the support for Safari on iOS in caniuse?

It true for most of features.

> For Edge and Chrome in Android, is it safe to assume that I can look at the support for Chrome for Android in caniuse?

It is not 100% safe. Edge could have changed in engine.

**Reply by NansySalem** [reply]
Wow, that was quick. I actually just edited my question. Would you please have another look? 

Also follow up question, when you say it is true for most features, what do you mean? Also what do you mean that Edge could have changed in Engine?

**Reply by ai** [reply]
> Should I be providing an edge/chrome version or a safari version?

Provide where?

**Reply by NansySalem** [reply]
I mean in the browser support matrix, which number should be provided an edge/chrome version or a safari version?

**Reply by ai** [reply]
> Is there internal mapping, for example if I'd like my app to support an older Edge version where mobile versioning was different from desktop, will this automatically map to a safari version automatically and vice versa?

No.

But we don't recommend to using queries like `BROWSER > VERSION`.

You should use queries like `last 2 versions` or `>= 1%`.

Use policies, not versions

**Reply by ai** [reply]
I don't understand why you should look to matrix before defining Browserslist config.

What exactly is your company's policy of supporting the browsers?

**Reply by NansySalem** [reply]
I understand it is not recommended to use version; however, this is the requirement and hence my question. It looks like I confused you, I mean for example here:

"browserslist": [
		"ios_saf >= what number should go here? edge/chrome mobile version? or, iOS Safari version?",
	],

**Reply by ai** [reply]
You should put there Safari version.

But it is a bad practice to have `ios_saf >= X`
